### PR TITLE
Update client

### PR DESCRIPTION
**Pull request by jonnywildey**
the client wasn't actually exporting anything, which is a problem if you're using it with typescript.

Also updated deps, which for some reason meant I had to change how some imports and mocks worked???

### CODE REVIEW

File: src/snowy.spec.ts [hunk 1]
```diff
@@ -8,18 +8,16 @@ const mockLedgersPost = jest.fn().mockResolvedValue(postResourceId);
...
-jest.mock("./ledgers", () => jest.fn().mockImplementation(() => mockLedgers));
+import Ledgers from "./ledgers";
+Ledgers.prototype.post = mockLedgersPost;
+Ledgers.prototype.put = mockLedgersPut;
 
 const mockEmptyReturn = jest.fn().mockResolvedValue({});
-const mockContents = { post: mockEmptyReturn };
-jest.mock("./contents", () => jest.fn().mockImplementation(() => mockContents));
+import Contents from "./contents";
+Contents.prototype.post = mockEmptyReturn;
```
Comment (chiaratrussle) [inline src/snowy.spec.ts]: Why did you change the way these are mocked?

Comment (jonnywildey) [reply src/snowy.spec.ts]: they weren't working, unfortunately. I'm guessing jest subtly changed how mockImplementation worked with classes
